### DISCUSSION TITLE
Regional Lora

### DISCUSSION THREAD

**Discussion by kantsche** [Acly/krita-ai-diffusion#904] [Ideas]
This might be a bit harder but could be nice with the new regional UI.

The way it could be done for an eternity was with the impact pack, but I dont know if there is a better way.
https://github.com/ltdrdata/ComfyUI-extension-tutorials/blob/Main/ComfyUI-Impact-Pack/tutorial/regional_sampler.md

**Comment by Acly** [comment]
I don't think it's possible with the latent masks region approach. What you linked is more similar to automated inpainting of each region. It doesn't yield coherent composition in my experience.

What already works is adding your Lora to region prompts. If you generate the full image, Lora of all regions will apply - but when you use the inpaint mode only the selected region's Lora are used.

**Reply by lobastik** [reply]
Can you please describe what you mean exactly? Maybe i'm doing it wrong but when i call lora in regional prompt (ex: <lora:NAME OF THE LORA:0.8>) it used even when generate full image and it took a lot time. When i use all selected loras without adding them to regions full image gen can take like 30 secs, but then i use these loras in regional prompts it took around 3-4 mins. is that okay? (as i can see while generating i still have spare 2 gigs of VRAM and enough RAM)
P.S. sorry for eng and if it's wrong thread to ask.

**Reply by Acly** [reply]
It will be used when you generate the full image, but only apply to the region.

Yes it can be very slow unless you have enough spare VRAM to hold the main model multiple times (eg. like 8GB free on top of normal use for SDXL)
